index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Hello</title>
+</head>
+<body>
+    <h1 id = title>"Hello Microverse"</h1>
```

Comment:
[OPTIONAL] kindly wrap id with " ", so it will be like:
```
 <h1 id ="title">Hello Microverse</h1>
```
For valid HTML syntax.